### ISSUE TITLE
"go get -u" started failing

### ISSUE THREAD

**Issue by shlomif** [tdewolff/minify#290]
```
shlomif[perl-begin]:$trunk$ go get -u github.com/tdewolff/minify/cmd/minify
# github.com/tdewolff/minify/svg
/home/shlomif/apps/golang/src/github.com/tdewolff/minify/svg/buffer.go:42:70: undefined: xml.EntitiesMap
/home/shlomif/apps/golang/src/github.com/tdewolff/minify/svg/svg.go:85:64: undefined: xml.EntitiesMap
```

with golang-1.13.2 on both local mageia x64 v8 and remote travis-ci ubuntu 18.04 x64.

Please look into fixing it.

**Comment by tdewolff** [comment]
Fixed by 06ce08abae174f636c518b9f39e499624413b577

**Comment by shlomif** [comment]
@tdewolff : thanks for the quick fix! It seems to be working fine now.
